src/index.html

```diff
@@ -3,12 +3,77 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="navList">
+          <li class="navItem">
+            <a
+              href="#"
+              class="navLink is-active"
+              >Apple
+            </a>
```

Comment:
There is no need to place each attribute to a newline if there's less than 3 attributes, you can fix it everywhere
```suggestion
            <a href="#" class="navLink is-active">
            	Apple
            </a>
```